client/me/security-2fa-key/delete-item-button.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { successNotice } from '../../state/notices/actions';
+import { recordGoogleEvent } from '../../state/analytics/actions';
+import Dialog from 'components/dialog';
+import Gridicon from 'gridicons'; // eslint-disable-line no-restricted-imports
+
+class Security2faKeyDeleteButton extends Component {
+	static propTypes = {
+		securityKey: PropTypes.object.isRequired,
+	};
+
+	state = {
+		showDialog: false,
+	};
+	buttons = {};
+
+	handleRemoveKeyButtonClick = () => {
+		this.buttons = [
+			{ action: 'cancel', label: this.props.translate( 'Cancel' ) },
+			{
+				action: 'delete',
+				label: this.props.translate( 'Remove key' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Remove Key' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).